docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+services:
+  nuxt:
+    image: kozr/lhd2020_nuxt
```

Comment:
We probably want to be able to rebuild the container in case we add more node modules, we should use build instead of a preset image.